apps/dashboard/src/components/workflow-editor/configure-workflow-form.tsx

```diff
@@ -197,6 +188,10 @@ export const ConfigureWorkflowForm = (props: ConfigureWorkflowFormProps) => {
                     <FormControl className="text-xs text-neutral-600">
                       <TagInput
                         {...field}
+                        onChange={(tags) => {
+                          form.setValue('tags', tags, { shouldValidate: true, shouldDirty: true });
+                          flushFormUpdates();
+                        }}
```

Comment:
One of the things that I'd like to be DRYers is this. It will get repetitive to add this logic on every field that doesn't require a blur. Maybe it's better to do `useFormAutosave()` with a 0 sec delay that will only affect specified fields (which will be those who don't update on blur).

Reply:
yes, but this creates another issue of independent forms (which in the UI is one form), and if one has the validation error, the other one should not be allowed to submit